modules/core/rapid_context.js

```diff
@@ -121,12 +121,23 @@ export function coreRapidContext(context) {
             id: 'msBuildings',
             beta: false,
             added: true,         // whether it should appear in the list
-            enabled: true,       // whether the user has checked it on
+            enabled: false,       // whether the user has checked it on
             conflated: true,
             service: 'fbml',
             color: RAPID_MAGENTA,
             label: t('rapid_feature_toggle.msBuildings.label'),
             license_markdown: t('rapid_feature_toggle.msBuildings.license_markdown')
+          },
```

Comment:
I love, love, love how @bhousel  data-fied the layers in this fashion. Makes adding a new feature layer a snap.